src/main/java/com/griddynamics/pift/EntityManager.java

```diff
@@ -26,6 +27,60 @@ public void flush() {
         createdEntitiesList.clear();
     }
 
+    public <T> List<T> getList(T entity) {
+        Class<T> type = (Class<T>) entity.getClass();
+        EntityUtils.checkOnTable(type);
+        String queryForSelect = SQLUtils.createQueryForSelect(entity);
+        List<T> entityList = new ArrayList<>();
+        try (Connection con = DriverManager.getConnection(URL, USER, PASSWORD);
+             Statement stmt = con.createStatement();
+             ResultSet rs = stmt.executeQuery(queryForSelect)
+        ) {
+            log.info(queryForSelect);
+            while (rs.next()) {
+                entityList.add(getEntityFromResultSet(type, rs));
+            }
+            return entityList;
+        } catch (Exception e) {
+            throw new IllegalStateException("Error in SQL", e);
+        }
+    }
+
+    public <T> Optional<T> getById(Class<T> type, Object id) {
+        EntityUtils.checkOnTable(type);
+        String query = SQLUtils.createQueryForSelectById(type, id);
+        log.debug(query);
+        try (Connection con = DriverManager.getConnection(URL, USER, PASSWORD);
+             Statement stmt = con.createStatement();
+             ResultSet rs = stmt.executeQuery(query)
+        ) {
+            if (rs.isBeforeFirst()) {
+                rs.next();
+                return Optional.of(getEntityFromResultSet(type, rs));
+            } else return Optional.empty();
+        } catch (Exception e) {
+            throw new IllegalStateException("Error in SQL", e);
+        }
+    }
+
+    private <T> T getEntityFromResultSet(Class<T> type, ResultSet resultSet) {
+        T entityInstance = ReflectionUtils.createInstance(type);
+        ReflectionUtils.getColumnFields(type).forEach(field -> {
+            try {
```

Comment:
Let's create separate method in EntityUtils: setField(Class<?>, ResultSet, Field) 
And move getEntityFromResultSet in EntityUtils